README.md

```diff
@@ -1,8 +1,125 @@
 # python-seed
 
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/10407788/90814089-905bf800-e2f6-11ea-8bd7-2639fa1177af.png" style="max-width: 800px;" alt="python-seed"></a>
+</p>
+<p align="center">
+  <em>Starter kit for creating a new python package.</em>
+</p>
+<p align="center">
+  <a href="https://circleci.com/gh/developmentseed/python-seed" target="_blank">
+      <img src="https://circleci.com/gh/developmentseed/python-seed.svg?style=svg" alt="Test">
+  </a>
+  <a href="https://codecov.io/gh/developmentseed/python-seed" target="_blank">
+      <img src="https://codecov.io/gh/developmentseed/python-seed/branch/master/graph/badge.svg" alt="Coverage">
+  </a>
+  <a href="https://pypi.org/project/python-seed" target="_blank">
+      <img src="https://img.shields.io/pypi/v/python-seed?color=%2334D058&label=pypi%20package" alt="Package version">
+  </a>
+  <a href="https://github.com/developmentseed/python-seed/blob/master/LICENSE" target="_blank">
+      <img src="https://img.shields.io/github/license/developmentseed/python-seed.svg" alt="Downloads">
+  </a>
+</p>
+
 This is a starter repo for creating a new python package. Included are templates for standard files as well as best practices.
 
-Update the setup.py file to reflect the proper PACKAGENAME and rename the PACKAGENAME to the name of the package
+## Install
+
+You can install python-seed using pip
+
+```bash
+$ pip install -U pip
+$ pip install python-seed
+```
+
+or install from source:
+
+```bash
+$ git clone https://github.com/developmentseed/python-seed.git
+$ cd python-seed
+$ pip install -U pip
+$ pip install -e .
+```
+
+## Usage
+
+```bash
+$ pyseed --help
+Usage: pyseed [OPTIONS] COMMAND [ARGS]...
+
+  python-seed subcommands.
+
+Options:
+  --version  Show the version and exit.
+  --help     Show this message and exit.
+
+Commands:
+  create  Create new python seed skeleton
+```
+
+```
+$ pyseed create --help
+Usage: pyseed create [OPTIONS] NAME
+
+  Create new python seed skeleton.
+
+Options:
+  --help  Show this message and exit.
+```
+
+Create a new python project
+
+```bash
+# Create a project
+$ pyseed create awesomepythonproject
+
+# List files created
+$ ls -1 awesomepythonproject
+.circleci/
+.pre-commit-config.yaml
+README.md
+awesomepythonproject/
+requirements-dev.txt
+requirements.txt
+setup.py
+tests/
+tox.ini
+```
+
+# Project structure
+
+```
+my-project/
+ ├── .circleci/                  - CircleCI configuration.
+ ├── .pre-commit-config.yaml     - pre-commit configuration.
+ ├── README.md                   - project readme.
+ ├── my_project/                 - core python module.
+ ├── tests/                      - tests suite placeholder for your module.
+ ├── requirements.txt            - python requirements (!!! by default requirements are written in setup.py)
```

Comment:
what does 
> !!! by default requirements are written in setup.py
mean?

Reply:
by default, the requirements for `{pyseed}` is hardcoded in setup.py. We also provide requirements.txt and requirements-dev.txt for development purposes https://stackoverflow.com/questions/43658870/requirements-txt-vs-setup-py